src/en/guide/deployment/configuration.md

```diff
@@ -179,6 +179,10 @@ app.update_config(MyConfig())
 | WEBSOCKET_READ_LIMIT      | 2^16                  | High-water limit of the buffer for incoming bytes                                   |
 | WEBSOCKET_WRITE_LIMIT     | 2^16                  | High-water limit of the buffer for outgoing bytes                                   |
 
+::: tip FYI
+The `USE_UVLOOP` value will be ignored if running with Gunicorn.
```

Comment:
Should the Windows note be moved down here too?

Reply:
Sure! Thought about it, but since in the code it can be overriden with the config value I figured I'd put it up there with the default. But no worries, I'll move it back down tomorrow hopefully.